python/py-mercurial_extension_utils/Portfile

```diff
@@ -15,14 +15,19 @@ supported_archs     noarch
 license             BSD
 maintainers         nomaintainer
 
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
```

Comment:
what's the point of using the gitlab PG and then using PyPI as the `master_sites`? Is there a reason why you cannot pick one of the two approaches?

Reply:
The build failed due to changed distfile checksums, so I figured getting the binary fro PyPI was more stable.

Reply:
I still think one should not mix and match in this case; pick one approach (i.e., either gitlab or PyPI) and stick with it. If there is a checksum mismatch you can treat it as [a stealth update](https://trac.macports.org/wiki/PortfileRecipes#stealth-updates).